src/Application/Common/Results/ErrorCode.cs

```diff
@@ -35,6 +35,7 @@ public enum ErrorCode
     ItemNotBuyable,
     ItemNotFound,
     ItemNotOwned,
+    ItemNotReforgeable,
```

Comment:
I've missed that but I think this requires a migration.

Reply:
done